blocks/rich-text/index.js

```diff
@@ -587,6 +618,13 @@ export default class RichText extends Component {
 				if ( event.shiftKey || ! this.props.onSplit ) {
 					this.editor.execCommand( 'InsertLineBreak', false, event );
 				} else {
+					// For small screens, save the root node rectangle so it can
+					// the position can be scrolled to in the next focussed
+					// instance.
+					if ( window.innerWidth < 784 ) {
+						scrollPosition = this.editor.getBody().getBoundingClientRect();
```

Comment:
This is probably a left over?

Reply:
What do you mean with left over? :)

Reply:
Never mind, I though it was a local variable